src/transformStateWithClones.js

```diff
@@ -7,7 +7,40 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const resultArray = [];
```

Comment:
It is better to choose more meaningful names of the changed, for example: `transformedState`, `stateClone` etc.